src/GitTools.IssueTrackers/AuthSettings.cs

```diff
@@ -0,0 +1,38 @@
+namespace GitTools.IssueTrackers
+{
+    using System;
+    using JetBrains.Annotations;
+
+    public class AuthSettings
+    {
+        public AuthSettings()
+        {
+            IsEmpty = true;
+        }
+
+        public AuthSettings([NotNull] string username, [NotNull] string password)
```

Comment:
Having different auth contexts is a bad idea. Isn't that the whole point of GitTools.Core?